src/AppBanner/index.tsx

```diff
@@ -98,27 +64,38 @@ export const AppBanner = (props: Props): ReactElement | null => {
         return null;
     }
 
+    const onCloseClick: OnCloseClick = (evt, internalButtonId) => {
+        evt.preventDefault();
+        onCloseAction(internalButtonId);
+    };
+
+    const onCloseAction = (internalButtonId: number): void => {
+        setShowBanner(false);
+        document.body.focus();
+        trackClick({
+            internalButtonId,
+            ophanComponentId: ophanComponentId as string,
+        });
+    };
```

Comment:
I think these aren't used currently? (But I think they should be! See comment about passing an aggregated callback through to `BannerCloseButton` instead of more granular `trackClick` and `setShowBanner`).

Reply:
They're being used by both App Banner CTAs - they're separate from the the top-right "close" button but, unlike any other CTA button they actually do close the banner! These functions have been removed in their entirety from all the other banners and localised into the BannerCloseButton component ... does that make sense?

<img width="629" alt="Screenshot 2024-02-09 at 17 19 36" src="https://github.com/guardian/braze-components/assets/5357530/27d8c0ca-10ea-4904-b611-05418841acd5">

Reply:
Ah I see, thanks for explaining 👍